static/style/style.css

```diff
@@ -579,6 +579,9 @@
     font-size: 12px;
     letter-spacing: 0.5px;
   }
+  #filters input[type="text"].hidden {
+    visibility: hidden;
```

Comment:
```suggestion
    display: none;
```

Reply:
I did this because it will not create an empty space in the DOM, which seems to make more sense for this purpose.